types/index.d.ts

```diff
@@ -210,7 +210,7 @@ export function getItems(context: HookContext): any; // any[] | any | undefined;
  * Check which transport provided the service call.
  * {@link https://feathers-plus.github.io/v1/feathers-hooks-common/index.html#IsProvider}
  */
-export function isProvider(...transports: TransportName[]): PredicateFn;
+export function isProvider(...transports: TransportName[]): boolean;
```

Comment:
this is not correct, it should return "SyncPredicateFn", would you also please fix the test?